sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -1,10 +1,7 @@
-from openmoc import *
+import openmoc
 import openmoc.log as log
 import openmoc.plotter as plotter
```

Comment:
Sure, I removed the various submodule imports from all sample input files in place of a single ``import openmoc``. Note that this change greatly increases the size of the diff for this PR, though much of it will be relatively straightforward to review.